core/src/test/java/org/pentaho/platform/dataaccess/datasource/api/DatasourceServiceTest.java

```diff
@@ -90,7 +108,7 @@ public void shouldAllowAccessForAdminWithAllPermissions() throws ObjectFactoryEx
   }
 
   @Test
-  public void shouldNotAllowAccessForAdminWithoutPublishPermission() throws ObjectFactoryException {
```

Comment:
Easier to just use `throws Exception`. One less import. Not sure what advantage being this specific in exceptions, when exceptions shouldn't be thrown here